### PR TITLE
Fix threshold

### PR DESCRIPTION
**Pull request by smialko**
Sth happened with the previous PR so I'm creating a new one with fixes from Pawełs CR.

### CODE REVIEW

File: AirCasting/MicrophoneSession/MicrophoneManager.swift [hunk 1]
```diff
@@ -120,7 +120,8 @@ private extension MicrophoneManager {
                                        thresholdHigh: 80,
                                        thresholdMedium: 70,
                                        thresholdLow: 60,
-                                       thresholdVeryLow: 20)
+                                       thresholdVeryLow: -100)
+        #warning("Change thresholdVeryLow to 20")
```
Comment (jankrzempek) [inline AirCasting/MicrophoneSession/MicrophoneManager.swift:124]: Could You add ToDo: ->  #warning("ToDo: Change thresholdVeryLow to 20")

File: AirCasting/SessionViews/SessionCartView.swift [hunk 2]
```diff
@@ -83,46 +83,39 @@ private extension SessionCartView {
...
+                ChartView(stream: selectedStream,
+                          thresholds: thresholds)
+                    .frame(height: 200)
+            }
         }
     }
     
-    func displayButtons(threshold: SensorThreshold) -> some View {
+    func displayButtons(thresholds: [SensorThreshold]) -> some View {
         HStack(spacing: 20){
```
Comment (jankrzempek) [inline AirCasting/SessionViews/SessionCartView.swift:113]: HStack(spacing: 20){   ->   HStack(spacing: 20) { please add space 😇

**Review by jankrzempek — APPROVED** [review]
👌🏻👌🏻👌🏻

File: AirCasting/MicrophoneSession/MicrophoneManager.swift (hunk 1 above)
Comment (smialko) [reply AirCasting/MicrophoneSession/MicrophoneManager.swift:124]: Sure!

File: AirCasting/SessionViews/SessionCartView.swift (hunk 2 above)
Comment (smialko) [reply AirCasting/SessionViews/SessionCartView.swift:113]: Sure, thanks!

File: AirCasting/SessionViews/SessionCartView.swift [hunk 3]
```diff
@@ -29,26 +29,26 @@ struct SessionCartView: View {
     var showChart: Bool {
         !isCollapsed && session.type == .mobile && session.status == .RECORDING
     }
+    var hasStreams: Bool {
+        return session.allStreams != nil || session.allStreams != []
```
Comment (pawelgil) [inline AirCasting/SessionViews/SessionCartView.swift:33]: `return` is not needed for one-line functions

**Review by pawelgil — APPROVED** [review]

**Pull request merged**